app/controllers/transition_defence_requests_controller.rb

```diff
@@ -34,6 +34,7 @@ def transition_params
       defence_request: defence_request,
       transition_to: requested_transition,
       user: current_user,
+      auth_token: session[:user_token]
```

Comment:
How do you feel about passing the instantiated api client down from the controller? Injecting the dependency that way can stop us referencing a constant all the way down into the transition.